api/data/it/PI.tsx

```diff
@@ -357,7 +357,7 @@ export const CustomInfoblockImprese = () => {
                 target="_blank"
                 rel="noopener noreferrer"
                 aria-label="poste.it"
-                style={{ fontWeight: "bold", color: "primary.main" }}>
+                style={{ fontWeight: "bold", color: "#0073E6" }}>
```

Comment:
@salvatorediocaro why remove primary.main? is the same color

Reply:
primary.main doesn't work, the color is different, but with hexadecimal it doesn't cause problems

Reply:
@salvatorediocaro I realise now that this block is no more requested. It can be removed in toto or it's fine to have hex color